docs/server/source/drivers-clients/http-client-server-api.rst

```diff
@@ -46,8 +46,11 @@ Transactions
 
    Get the transaction with the ID ``tx_id``.
 
-   This endpoint returns a transaction only if a ``VALID`` block on
-   ``bigchain`` exists.
+   This endpoint returns a transaction if it was included in a ``VALID`` block,
+   if it is still waiting to be processed (``BACKLOG``) or is still in an
+   undecided block (``UNDECIDED``). All instances of a transaction in invalid
+   blocks are ignored and treated as if they don't exist. A request to an
+   invalid transaction will return a ``404 Not Found``.
```

Comment:
This last sentence bothers me because it's not clear what you mean by an "invalid transaction." The mere fact that a transaction is in a decided-invalid block doesn't mean that it's an invalid transaction. Maybe say:

If a request is made for a transaction and copies of that transaction are found only in invalid blocks, then the response will be `404 Not Found`.